examples/next/pages/ui/components/ai/ai-conversation/index.page.tsx

```diff
@@ -13,6 +13,13 @@ const { useAIConversation } = createAIHooks(client);
 
 Amplify.configure(outputs);
 
+const formatDate = (date) =>
```

Comment:
Non-blocking, could we type this function to make sure the types are working as expected?

Reply:
yep, will add